.github/workflows/changelog-generation.yml

```diff
@@ -26,14 +26,27 @@ jobs:
         run: |
             ./generate-changelog.sh
             cat CHANGELOG.md
-           
+
+      - name: Read Tag Name from VERSION File
+        id: tag_name
+        run: |
+          echo "tag_name=$(cat VERSION)" >> $GITHUB_ENV
+
+      - name: Get Base SHA from Tag
+        id: base_sha
+        run: |
+          echo "base_sha=$(git rev-list -n 1 "v${{ env.tag_name }}")" >> $GITHUB_ENV
+        continue-on-error: true
+
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v6
         with:
-          commit-message: "chore: autogenerate changelog"
-          title: "Changelog"
-          body: This PR contains an autogenerated changelog for newer commits. The new changelog can be modified [here](https://github.com/prometheus/client_golang/blob/changelog-branch/CHANGELOG.md).
+          commit-message: "chore: cut ${{ github.ref_name }}"
+          title: "Cut ${{ github.ref_name }}"
+          body: |
+            This PR contains an autogenerated changelog for newer commits. 
+            You can view the list of commits made after the release of version v${{ env.tag_name}} [here](https://github.com/${{ github.repository }}/compare/${{ env.base_sha }}...main)
```

Comment:
Do we need to add `/kind NONE` to the body message, so it passes the auto-label workflow that we're adding in #1518 ?

Reply:
Wow, thanks for pointing this out! 🔥
You're right, the PR description should adhere to the PR template to prevent any future issues.